src/c++/perf_analyzer/main.cc

```diff
@@ -965,6 +999,26 @@ main(int argc, char** argv)
         measurement_request_count = std::atoi(optarg);
         break;
       }
+      case 28: {
+        std::string arg = optarg;
+        if (!arg.empty()) {
```

Comment:
You can just set server_library_path = optarg as is done in the other arg parsing. I don't understand the need for the error message.

Reply:
Still needs to be fixed.